introduction/E_installation.md

```diff
@@ -45,6 +45,24 @@ We can get node.js from the [download page](https://nodejs.org/download/). When
 
 Mac OS X users can also install node.js via [homebrew](http://brew.sh/).
 
+#### Debian/Ubuntu users
+You might see an error that says something like:
+```console
+sh: 1: node: not found
+npm WARN This failure might be due to the use of legacy binary "node"
+```
+This due to Debian having conflicting binaries for node: [discussion on stackoverflow](http://stackoverflow.com/questions/21168141/can-not-install-packages-using-node-package-manager-in-ubuntu)
+
+There are two options to this:
+- installing nodejs-legacy:
```

Comment:
If we do the edit from the previous line, then "installing" should become "install"